### DISCUSSION TITLE
Slå sammen spacingverdier

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2375] [Annet]
Etter litt debatt på Forum rundt bruken og dokumentasjonen av spacingverdiene våre, ble vi enige om at det ikke var noe særlig nytteverdi for oss å skille mellom komponent- og layoutverdier. Per i dag bruker vi t-skjorte størrelser _(xxs, xs, s, m, osv osv)_. Slik jeg har forstått det på den faktiske implementasjonen i Jøkul så varierer dette mellom forkortelser av størrelsen og hele ordet.

Såvidt jeg husker så hadde @Saegrov noen meninger om hvordan vi burde navngi selve klassene? Fint om vi deler litt innsikt og eventuelle andre meninger her. Jeg kan fyre opp et issue og ta meg av Figma-dokumentasjonen når vi er i boks.

**Comment by onordbo** [comment]
Som designer føler jeg ikke at jeg påvirkes nevneverdig av selve navngivningen, men hvis jeg skulle kastet min stemme til forkortelser vs full-lengde størrelsesnavn så lener jeg nok mer mot forkortelser, av rene praktiske årsaker. Det er forøvrig litt mindre "verbose" _(finnes det ett godt norsk ord for det eller?)_ 😂

**Comment by onordbo** [comment]
### Oppdatering fra Designsystem-forum:
Vi går for tskjortestørrelser i forkortet stil. Etter _dobbel-xl/xs_ legger vi til tall for størrelsesmultiplikasjon. Legg gjerne til ekstra detaljer hvis jeg glemte noe @Mikaila94.
> Eksempel klasse: `.jkl-spacing-#{verdi}--#{all | top | bottom | left | right}`

![image](https://user-images.githubusercontent.com/4263132/136957064-5328ce32-93d8-4c57-a3b9-1211318f191f.png)

**Reply by wkillerud** [reply]
Vi ble også enige om å gå for noe a la `2xl`, `3xl`og `4xl` (tilsvarende på `xs`-siden) for lesbarhet. Litt enklere å se hvor mange `x` det er snakk om enn en lang remse med `xxxx`.
